acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/PublicKeysOpenApiValidationTest.java

```diff
@@ -44,12 +43,18 @@
   @TempDir Path testDirectory;
 
   @Test
-  public void noLoadedKeysReturnsEmptyPublicKeyResponse() throws Exception {
+  public void noLoadedKeysReturnsEmptyPublicKeyResponse() {
     startSigner(new SignerConfigurationBuilder().build());
 
-    final HttpResponse response = signer.getRawRequest("/signer/publicKeys");
-    assertThat(response.getStatusCode()).isEqualTo(HttpResponseStatus.OK.code());
-    assertThat(publicKeysFromResponse(response)).isEmpty();
+    given()
+        .baseUri(signer.getUrl())
+        .filter(getOpenApiValidationFilter())
+        .when()
+        .get(SIGNER_PUBLIC_KEYS_PATH)
+        .then()
+        .assertThat()
```

Comment:
Does make sense split out this request code? Looks like the only difference between functions is the expected response